qemu/tests/balloon_thp.py

```diff
@@ -0,0 +1,64 @@
+import time
+import logging
+
+from virttest import funcatexit
+from virttest.staging import utils_memory
+
+
+def clean_env(session, file_name):
+    """
+    Clean the file created inside guest
+    :param session: The vm session
+    :param file_name: The file name
+    """
+    session.cmd_output_safe('rm -rf %s' % file_name)
+
+
+def run(test, params, env):
+    """
+    Memory balloon with thp
+
+    1. Boot up a guest with balloon support, record memory fragement
+    2. Make fragement in guest with tmpfs
+    3. check the memory fragement with proc system, should increase
+    4. Do memory balloon the memory size ballooned should be a legal value
+    5. Check the memory fragement with proc system
+    6. Try to make the memory compact
```

Comment:
After a second thought, I think this step has nothing to with virt, so I suggest to remove step 6~8. @MiriamDeng Do you agree?

Reply:
@huangyum Yes, we can remove them. Thanks.

Reply:
@huangyum @MiriamDeng Updated and remove those steps, but final step is `Check the memory fragement with proc system`, only print the result without any comparison?

Reply:
No, we need compare the result. I rem it's there before?

Reply:
@huangyum It compares with the result after step 7, but now step 7 is removed, which result it should compare with now? Step 3? And should it increase or decrease?

Reply:
Yes, compared to step3, should decrease.

Reply:
@huangyum Updated, thanks.